modules/apps/object/object-web/types/src/main/resources/META-INF/resources/js/components/ObjectField/SearchableContainer.d.ts

```diff
@@ -24,7 +24,6 @@ interface ISearchableProps {
 	setValues: (values: Partial<ObjectField>) => void;
 }
 export declare function SearchableContainer({
-	disabled,
```

Comment:
@GabrielHenriquedev Please, squash this commit with the next one. This change only makes sense when it's together with the change in the component.

Reply:
Okay, thanks for the review.